src/platform/graphics/webgl/webgl-graphics-device.js

```diff
@@ -613,44 +667,46 @@ class WebglGraphicsDevice extends GraphicsDevice {
                 uniformValue[2] = scopeZ;
             }
         };
-        this.commitFunction[UNIFORMTYPE_VEC4]  = function (uniform, value) {
+        this.commitFunction[UNIFORMTYPE_IVEC3]  = function (uniform, value) {
```

Comment:
personally I'd avoid changing / reordering things in the same PR as adding new functionality, as that makes it tricky to make sure all is correct.